cst438_gradebook/src/main/java/com/cst438/services/RegistrationServiceREST.java

```diff
@@ -1,26 +1,34 @@
 package com.cst438.services;
 
 import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.*;
 import org.springframework.web.client.RestTemplate;
 
 import com.cst438.domain.CourseDTOG;
 
+import java.net.URI;
+import java.net.URISyntaxException;
+
 public class RegistrationServiceREST extends RegistrationService {
+    RestTemplate restTemplate = new RestTemplate();
+
+    @Value("${registration.url}")
+    String registration_url;
+
+    public RegistrationServiceREST() {
+        System.out.println("REST registration service ");
+    }
+
+    @Override
+    public void sendFinalGrades(int course_id, CourseDTOG courseDTO) throws URISyntaxException {
```

Comment:
This was added as a new public statment. sendFinalGrades included the strings course_id with the CourseDTOG class. 
Addition to that helps out with the System.out.printIn to create a function for the application to add final grades + course_id + courseDTO"